src/PrestaShopBundle/Bridge/AdminController/FrameworkBridgeControllerListTrait.php

```diff
@@ -29,71 +29,66 @@
 namespace PrestaShopBundle\Bridge\AdminController;
 
 use PrestaShopBundle\Bridge\AdminController\Action\ActionInterface;
-use PrestaShopBundle\Bridge\AdminController\Action\HeaderToolbarAction;
 use PrestaShopBundle\Bridge\AdminController\Action\ListBulkAction;
 use PrestaShopBundle\Bridge\AdminController\Action\ListHeaderToolbarAction;
 use PrestaShopBundle\Bridge\AdminController\Action\ListRowAction;
 use PrestaShopBundle\Bridge\AdminController\Field\FieldInterface;
 use PrestaShopBundle\Bridge\Exception\NotAllowedActionTypeForListException;
-use PrestaShopBundle\Bridge\Exception\NotAllowedGenericActionTypeException;
 use PrestaShopBundle\Bridge\Helper\FiltersHelper;
 use PrestaShopBundle\Bridge\Helper\HelperListBridge;
 use PrestaShopBundle\Bridge\Helper\HelperListConfiguration;
 use PrestaShopBundle\Bridge\Helper\ResetFiltersHelper;
 
 /**
- * This trait contains the principal methods you need when you want to migrate a controller horizontally.
- *
- * This trait adds methods to:
- *     - add a general action
- *     - add a specific action to list
- *     - add a field to a controller
- *     - get reset filter helpers
- *     - get filter helpers
- *     - get helper list bridge
+ * Contains the principal methods you need to horizontally migrate a controller which has a list.
  */
-trait AdminControllerTrait
+trait FrameworkBridgeControllerListTrait
 {
+    public function buildListConfiguration(
```

Comment:
Since all these methods are meant to be used internally in the controller the should probably be `protected`
We don't need to do `$controller->buildListConfiguration(...)` from external code at some point, so no need to make these methods public, accessible and executable